.github/workflows/test-integration.yml

```diff
@@ -21,6 +21,8 @@ jobs:
           - { opensearch_version: 2.5.0, java: 11 }
           - { opensearch_version: 2.6.0, java: 11 }
           - { opensearch_version: 2.7.0, java: 11 }
+          - { opensearch_version: 2.8.0, java: 11 }
+          - { opensearch_version: 2.9.0, java: 11 }
```

Comment:
Just fyi, 2.10 is supposed to be released on Monday (:crossed_fingers: ), may be we could include it as well

Reply:
Definitely! I will leave this open till then.